Arm/Hand/wrist_joint.hpp

```diff
@@ -63,14 +63,14 @@ class WristJoint
     roll_offset_angle_ = roll_offset;
   }
 
-  /// Return the acceleration values for the MPU6050 on the joint.
+  //return is only used for testing
```

Comment:
If the return value is only used for testing and the acceleration_ member variable is public, then it seems redundant. You could make this function void or make acceleration_ private. imo keeping acceleration_ public is easier  since we can potentially use that to directly modify the acceleration values for other testing purposes

Reply:
we use the acceleration_ in the rover arm system, so I'll keep it public and make it a void function.